exopy_hqc_legacy/tasks/tasks/instr/run_awg.py

```diff
@@ -22,20 +24,45 @@ class RunAWGTask(InstrumentTask):
     """
     #: Switch to choose the AWG run mode: on or off
     switch = Unicode('Off').tag(pref=True, feval=validators.SkipLoop())
+    delay = Float(0).tag(pref=True,
```

Comment:
Since you are enforcing a Float using Atom, you do not need a feval validator, there it nothing to evaluate. I am even surprised this does not crash.